docs/configuration/mftf.md

```diff
@@ -15,6 +15,83 @@ SELENIUM_HOST=selenium-hub
 BROWSER=chrome
 ```
 
+Of note - `selenium-hub` can be unreliable, and you may have a better experience using `selenium-chrome`.
+
+To use `selenium-chrome`:
+
+If `.warden/warden-env.yml` doesn't exist, create it in the root of your project, and insert the following:
+
+```
+version: "3.5"
+services:
+  selenium-chrome:
+    hostname: ${WARDEN_ENV_NAME}_selenium-chrome
+    image: selenium/standalone-chrome${WARDEN_SELENIUM_DEBUG:-}:3.8.1
+    extra_hosts:
+      - ${TRAEFIK_SUBDOMAIN:-app}.${TRAEFIK_DOMAIN}:${TRAEFIK_ADDRESS:-0.0.0.0}
+    environment:
+      - START_XVFB=false
```

Comment:
@Lunaetic @lbajsarowicz Any reason this change couldn't/shouldn't be simply made in the Warden default environment spec for MFTF? Doesn't make sense to me to include a broken Selenium setup (even if it's not really our fault) and then have a doc saying how to override it with one that works.